reference/configuration.html.md.erb

```diff
@@ -291,15 +291,18 @@ Times are in milliseconds unless units are specified.
 
 ## The `services` sections
 
-The `services` sections configure the mapping of ports on the application to ports and services on Fly.io. These mappings determine how connections to the application will be handled on their journey from the Fly.io edge network to running Fly Apps.
+The `services` sections define how Fly Proxy connects requests to services running within Machines, via an app's public [Anycast](/docs/reference/services/) or private [Flycast](/docs/reference/private-networking/#flycast-private-load-balancing) address. They also configure additional settings tied to Fly Proxy behavior for a service. If a service only needs HTTP and HTTPS on standard ports, you can use the less verbose [`http_service`](#the-http_service-section).
 
-You can have:
+A `services` section itself is a table of tables in TOML, so it is delimited with double square brackets. Tables within `[[services]]` configure additional features and behaviors for the service, and are described in subsequent sections. **At least one [`services.ports`](#services-ports) entry is required for a complete service definition, to set a port and handler for incoming requests.**
 
-* No `services` section (and no [`http_service` section](#the-http_service-section)): The application has no mappings to the external internet - typically apps like databases that talk over [6PN private networking](/docs/reference/private-networking/) to other apps.
-* One `services` section (or an [`http_service` section](#the-http_service-section)): One internal port mapped to the one or more external ports on the internet.
+An app can have:
+
+* No `services` section (and no [`http_service` section](#the-http_service-section)): The application cannot be reached from the public internet, nor by a [Flycast](/docs/reference/private-networking/#flycast-private-load-balancing) address; this is typically apps like databases that talk over [6PN private networking](/docs/reference/private-networking/) to other apps.
+* One `services` section (or an [`http_service` section](#the-http_service-section)): One internal port mapped to one or more external ports.
 * Multiple `services` sections (or an [`http_service` section](#the-http_service-section) and one or more `services` sections): Mapping multiple internal ports to multiple external ports.
+  
 
-The `services` section itself is a table of tables in TOML, so the section is delimited with double square brackets, like this:
+Example (note the double square brackets):
 
 ```toml
```

Comment:
I wonder if the initial services example should include all the required sections? Or if we should have one complete example at the top and then break each piece out into the subsequent sections? And be more explicit about what's required and what's optional.

Reply:
this might be Out Of Scope 👾

Reply:
For this PR, I think so. 😁